Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
Exception: this is a message
